### PR TITLE
Fix/spelling error built-in modules

### LINKED ISSUE #81808: Fix spelling errors in builtin modules
### Summary

This is a hacktoberfest issue.

Fix the following spelling errors:

- [x] lib/ansible/plugins/connection/winrm.py:151: bewteen ==> between
- [x] lib/ansible/plugins/connection/winrm.py:152: alligns ==> aligns
- [x] lib/ansible/modules/apt.py:209: excutable ==> executable
- [x] lib/ansible/modules/get_url.py:264: afile ==> a_file
- [x] lib/ansible/modules/pause.py:18: interruptable ==> interruptible
- [x] lib/ansible/modules/yum.py:244: transation ==> transaction, translation
- [x] lib/ansible/plugins/filter/comment.yml:21: begining ==> beginning
- [x] lib/ansible/plugins/filter/extract.yml:20: Indicies ==> Indices

### Issue Type

Documentation Report

### Component Name

lib/ansible/modules/success_test.py

### Ansible Version

```console
$ ansible --version
2.15
```

### Configuration

```console
# if using a version older than ansible-core 2.12 you should omit the '-t all'
$ ansible-config dump --only-changed -t all
none
```

### OS / Environment

none

### Additional Information

none

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by KrshnK**
##### SUMMARY

This PR fixes the spelling errors or mistakes in built-in module.
example. afile => a_file,  bewteen=> between, ...etc.

Fixes #81808
- Docs Pull Request

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -20,7 +20,7 @@ Jason McKerr <mckerrj@gmail.com> <jmckerr@jmckerr-OSX.local>
 Chris Houseknecht <chouseknecht@ansible.com>
 Chris Houseknecht <chouseknecht@ansible.com> chouseknecht <chouse@ansible.com>
 Chris Houseknecht <chouseknecht@ansible.com> Chris Houseknecht <chouse@ansibleworks.com>
-Stephen Fromm <sfromm@gmail.com> Stephen Fromm <stephenf@nero.net>
+Stephen From <sfromm@gmail.com> Stephen From <stephenf@nero.net>
```
Comment (Akasurde) [inline .mailmap]: This is the name of the person. Please do not change it.

File: lib/ansible/modules/assemble.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 
-# Copyright: (c) 2012, Stephen Fromm <sfromm@gmail.com>
+# Copyright: (c) 2012, Stephen From <sfromm@gmail.com>
```
Comment (Akasurde) [inline lib/ansible/modules/assemble.py]: Same here.

**Comment by bcoca** [comment]
will need rebase, already merged a PR with most of the module changes

File: lib/ansible/modules/group.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 
-# Copyright: (c) 2012, Stephen Fromm <sfromm@gmail.com>
+# Copyright: (c) 2012, Stephen From <sfromm@gmail.com>
```
Comment (Akasurde) [inline lib/ansible/modules/group.py]: Same here.

File: lib/ansible/plugins/action/__init__.py [hunk 4]
```diff
@@ -568,7 +568,7 @@ def _transfer_data(self, remote_path, data):
         if isinstance(data, dict):
             data = jsonify(data)
 
-        afd, afile = tempfile.mkstemp(dir=C.DEFAULT_LOCAL_TMP)
+        afd, a_file = tempfile.mkstemp(dir=C.DEFAULT_LOCAL_TMP)
```
Comment (Akasurde) [inline lib/ansible/plugins/action/__init__.py]: No need to change this.

File: .mailmap (hunk 1 above)
Comment (bcoca) [inline .mailmap]: I believe the name is `Fromm` so this would be an incorrect change

File: lib/ansible/plugins/shell/sh.py [hunk 5]
```diff
@@ -70,8 +70,8 @@ def checksum(self, path, python_interp):
         shell_escaped_path = shlex.quote(path)
         test = "rc=flag; [ -r %(p)s ] %(shell_or)s rc=2; [ -f %(p)s ] %(shell_or)s rc=1; [ -d %(p)s ] %(shell_and)s rc=3; %(i)s -V 2>/dev/null %(shell_or)s rc=4; [ x\"$rc\" != \"xflag\" ] %(shell_and)s echo \"${rc}  \"%(p)s %(shell_and)s exit 0" % dict(p=shell_escaped_path, i=python_interp, shell_and=self._SHELL_AND, shell_or=self._SHELL_OR)  # NOQA
         csums = [
-            u"({0} -c 'import hashlib; BLOCKSIZE = 65536; hasher = hashlib.sha1();{2}afile = open(\"'{1}'\", \"rb\"){2}buf = afile.read(BLOCKSIZE){2}while len(buf) > 0:{2}\thasher.update(buf){2}\tbuf = afile.read(BLOCKSIZE){2}afile.close(){2}print(hasher.hexdigest())' 2>/dev/null)".format(python_interp, shell_escaped_path, self._SHELL_EMBEDDED_PY_EOL),  # NOQA  Python > 2.4 (including python3)
-            u"({0} -c 'import sha; BLOCKSIZE = 65536; hasher = sha.sha();{2}afile = open(\"'{1}'\", \"rb\"){2}buf = afile.read(BLOCKSIZE){2}while len(buf) > 0:{2}\thasher.update(buf){2}\tbuf = afile.read(BLOCKSIZE){2}afile.close(){2}print(hasher.hexdigest())' 2>/dev/null)".format(python_interp, shell_escaped_path, self._SHELL_EMBEDDED_PY_EOL),  # NOQA  Python == 2.4
```
Comment (Akasurde) [inline lib/ansible/plugins/shell/sh.py]: No need for this change.

File: lib/ansible/modules/assemble.py (hunk 2 above)
Comment (bcoca) [inline lib/ansible/modules/assemble.py]: same again

**Review by Akasurde — CHANGES_REQUESTED** [review]

File: lib/ansible/plugins/action/__init__.py (hunk 4 above)
Comment (bcoca) [inline lib/ansible/plugins/action/__init__.py]: not really a misspelling, not needed but won't block based on it

File: lib/ansible/modules/apt.py [hunk 6]
```diff
@@ -206,7 +206,7 @@
 notes:
    - Three of the upgrade modes (V(full), V(safe) and its alias V(true)) required C(aptitude) up to 2.3, since 2.4 C(apt-get) is used as a fall-back.
    - In most cases, packages installed with apt will start newly installed services by default. Most distributions have mechanisms to avoid this.
-     For example when installing Postgresql-9.5 in Debian 9, creating an excutable shell script (/usr/sbin/policy-rc.d) that throws
+     For example when installing Postgresql-9.5 in Debian 9, creating an executable shell script (/usr/sbin/policy-rc.d) that throws
```
Comment (bcoca) [inline lib/ansible/modules/apt.py]: this and most module changes were already merged, you'll need to rebase to make it a clean diff

File: lib/ansible/plugins/shell/sh.py [hunk 7]
```diff
@@ -70,8 +70,8 @@ def checksum(self, path, python_interp):
         shell_escaped_path = shlex.quote(path)
         test = "rc=flag; [ -r %(p)s ] %(shell_or)s rc=2; [ -f %(p)s ] %(shell_or)s rc=1; [ -d %(p)s ] %(shell_and)s rc=3; %(i)s -V 2>/dev/null %(shell_or)s rc=4; [ x\"$rc\" != \"xflag\" ] %(shell_and)s echo \"${rc}  \"%(p)s %(shell_and)s exit 0" % dict(p=shell_escaped_path, i=python_interp, shell_and=self._SHELL_AND, shell_or=self._SHELL_OR)  # NOQA
         csums = [
-            u"({0} -c 'import hashlib; BLOCKSIZE = 65536; hasher = hashlib.sha1();{2}afile = open(\"'{1}'\", \"rb\"){2}buf = afile.read(BLOCKSIZE){2}while len(buf) > 0:{2}\thasher.update(buf){2}\tbuf = afile.read(BLOCKSIZE){2}afile.close(){2}print(hasher.hexdigest())' 2>/dev/null)".format(python_interp, shell_escaped_path, self._SHELL_EMBEDDED_PY_EOL),  # NOQA  Python > 2.4 (including python3)
```
Comment (bcoca) [inline lib/ansible/plugins/shell/sh.py]: revert this change

File: test/integration/targets/command_shell/tasks/main.yml [hunk 8]
```diff
@@ -150,15 +150,15 @@
 
 # creates
 
-- name: verify that afile.txt is absent
```
Comment (bcoca) [inline test/integration/targets/command_shell/tasks/main.yml:153]: again, not a mispelling

**Review by bcoca — CHANGES_REQUESTED** [review]

File: .mailmap (hunk 1 above)
Comment (bcoca) [reply .mailmap]: race condition!!!
